esp/esp/dbmail/cronmail.py

```diff
@@ -45,7 +45,7 @@
 
 import os
 
-@transaction.autocommit
+@transaction.atomic
```

Comment:
Unlike the others, this and the other change to the same file are actual changes in behavior. I haven't taken the time to think about which one is what we want. Relevant: #1666